packages/edge-login-ui-rn/src/components/screens/existingAccout/SecurityAlertsScreen.js

```diff
@@ -67,9 +67,10 @@ export class SecurityAlertsScreenComponent extends React.Component<
   componentDidMount() {
     const { account } = this.props
     this.cleanups = [
-      account.watch('otpResetDate', otpResetDate =>
-        this.setState({ otpResetDate }, this.checkEmpty)
-      ),
+      account.watch('otpResetDate', otpResetDate => {
+        const clean = otpResetDate != null ? new Date(otpResetDate) : undefined
```

Comment:
You already called the param `clean`, why not actually use a cleaner? And you can use the same cleaner on line 58.

Reply:
A cleaner isn't appropriate, because this isn't random outside data. I will change the name to `date` instead.